pom.xml

```diff
@@ -78,9 +86,34 @@
     <url>https://github.com/hawkular/hawkular-commons</url>
     <tag>HEAD</tag>
   </scm>
+
   <ciManagement>
     <system>Travis-ci</system>
     <url>https://travis-ci.org/hawkular/hawkular-commons</url>
   </ciManagement>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.wildfly.bom</groupId>
+        <artifactId>jboss-javaee-7.0-wildfly</artifactId>
+        <version>${version.org.wildfly}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-spec-api</artifactId>
```

Comment:
This is a BOM that includes all specs used by Wildfly. I needed this for accounts, but it seems it's not needed for commons. So, I'm removing it.